cmd/pets/provider.go

```diff
@@ -0,0 +1,19 @@
+package pets
+
+import (
+	"os"
+
+	"github.com/windmilleng/pets/internal/mill"
+	"github.com/windmilleng/pets/internal/proc"
+	"github.com/windmilleng/pets/internal/school"
+)
+
+func newPetsitter() (*mill.Petsitter, error) {
```

Comment:
i poked around at using https://github.com/google/go-cloud/tree/master/wire to auto-generate this code, because it's pretty mechanical, but figured this is probably ok as-is